src/Microsoft.Performance.SDK.Runtime/DTO/PreV1/TableConfiguration.cs

```diff
@@ -110,64 +110,65 @@ internal class TableConfiguration
 
         public DTO.TableConfiguration Upgrade()
         {
-            var newColumnRoles = new Dictionary<DTO.Enums.ColumnRole, ColumnRoleEntry>(this.ColumnRoles.Count);
+            var newColumnRoles = new Dictionary<string, ColumnRoleEntry>(this.ColumnRoles.Count);
 
             foreach (var columnRolePair in this.ColumnRoles)
             {
                 var role = columnRolePair.Key;
                 var entry = columnRolePair.Value;
 
-                var newEntry = DTO.Enums.ColumnRole.Invalid;
+                string newEntry = null;
 
                 switch(role)
                 {
                     case ColumnRole.Invalid:
+                    case ColumnRole.CountColumnMetadata:
+                        newEntry = null;
+                        break;
+
+
                     case ColumnRole.StartThreadId:
                     case ColumnRole.EndThreadId:
                     case ColumnRole.HierarchicalTimeTree:
                     case ColumnRole.WaitDuration:
                     case ColumnRole.WaitEndTime:
-                        newEntry = Enums.ColumnRole.Invalid;
+                        newEntry = role.ToString();
```

Comment:
Can we add a comment here explaining why we're doing this and how we're no longer guaranteeing a column role is "known" by the driver?